roles/splunk_common/tasks/set_hec_port.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: Set HTTP Envent Collector (HEC) Port
+  ini_file:
+    dest: "{{ splunk.home }}/etc/system/local/inputs.conf"
+    section: http
+    option: "port"
+    value: "{{ splunk.hec_port }}"
```

Comment:
Might be possible to do this via CLI in `roles/splunk_common/tasks/set_as_hec_receiver.yml` so we don't need to write conf files. 

https://docs.splunk.com/Documentation/Splunk/8.0.2/Data/UseHECfromtheCLI

Reply:
ok, task removed